ananse/network.py

```diff
@@ -57,6 +58,8 @@ def __init__(
                 Include or exclude promoter peaks (<= TSS +/- 2kb) in network inference. (default: False)
             include_enhancer : bool
                 Include or exclude enhancer peaks (> TSS +/- 2kb) in network inference. (default: True)
+            necessary_output : bool
```

Comment:
I dislike `necessary_output` as a variable name. Not only because ~nesse~, ~necce~... ~necesa~. _IT_  is one of the hardest words to write in the English language, but its also long and subjective. `full_output` perhaps?

Reply:
Good suggestion, I'll change it (and then do a new pull request on the updated version? Or should I alter a pull request or something?)

Reply:
you can push to your fork, and it will automatically update the pull request :)